client/src/plus/deep-dives/sample-articles.tsx

```diff
@@ -445,9 +470,16 @@ export function PlanningForBrowserSupport() {
 }
 
 export function YourBrowserSupportToolkit() {
+  const { hash } = useLocation();
   const { locale } = useParams();
   const [showFullArticle, setShowFullArticle] = React.useState(false);
 
+  React.useEffect(() => {
+    if (hash) {
+      setShowFullArticle(true);
+    }
+  }, [hash]);
+
   return (
     <article className="deep-dive-article-container">
       <header className="main-heading-group heading-group">
```

Comment:
I don't know where to leave this, but ` .main-heading-group h1`  should use MDN Plus / Headers / H1 with `line-height: 100%;` so the text wraps closer 
![image](https://user-images.githubusercontent.com/44966503/129600248-eecf2751-0b23-47d4-84a3-0ae37911948b.png)